lldb/include/lldb/Core/PluginManager.h

```diff
@@ -212,6 +212,8 @@ class PluginManager {
   static Status SaveCore(const lldb::ProcessSP &process_sp,
                          lldb_private::SaveCoreOptions &core_options);
 
+  static llvm::SmallVector<llvm::StringRef> GetSaveCorePluginNames();
```

Comment:
It's rather uncommon to return a `SmallVector` by value. Usually, you return a `SmallVectorImpl` by reference (you can't by value because it's a polymorphic type) so that the caller can decide how many elements there are. Given that this isn't actually hot code, I would just return a plain `vector` (rather than giving this an output argument).